pkg/commands/process/settings/ruleValidator.go

```diff
@@ -0,0 +1,68 @@
+package settings
+
+import (
+	"bytes"
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/ghodss/yaml"
+	"github.com/xeipuuv/gojsonschema"
+)
+
+const SCHEMA_URL = "https://raw.githubusercontent.com/Bearer/bearer-rules/main/scripts/rule_schema.json"
```

Comment:
Its the easiest way to get the schema otherwise I think we will need to download the sourcecode from the latest release and extract it that way. 

Bear in mind that the schema is only loaded if a rule does not unmarshal for some reason so you only end up using the schema if a rule is totally broken. Since this is the case I think mains fine to use.